ui/bin/www

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+'use strict'
+
+const app = require('../app')
+const http = require('http')
+const port = normalizePort(process.env.PORT || '3000')
+const server = http.createServer(app)
+const logger = require('../utils/logging/logger')
+
+// listen on provided port, on all network interfaces
+app.set('port', port)
+server.listen(port)
+server.on('error', onError)
+server.on('listening', onListening)
+
+// normalize a port into a number, string, or false
+function normalizePort(val) {
+    const port = parseInt(val, 10)
+    if (isNaN(port)) return val
+    if (port >= 0) return port
+    return false
```

Comment:
Rather than return false, maybe just thrown an error. If the port isn't set up, it's gonna thrown an error down the line anyways.